Lib/fontmake/font_project.py

```diff
@@ -250,28 +260,109 @@ def build_ttfs(self, ufos, **kwargs):
         """Build OpenType binaries with TrueType outlines."""
         self.save_otfs(ufos, ttf=True, **kwargs)
 
-    def build_interpolatable_ttfs(self, ufos, **kwargs):
-        """Build OpenType binaries with interpolatable TrueType outlines."""
-        self.save_otfs(ufos, ttf=True, interpolatable=True, **kwargs)
+    @staticmethod
+    def _load_designspace_sources(designspace):
+        # set source.font attributes, but only load fonts once
+        masters = {}
+        for source in designspace.sources:
+            if source.path in masters:
+                source.font = masters[source.path]
+            else:
+                assert source.path is not None
+                source.font = Font(source.path)
+                masters[source.path] = source.font
+
+    def _build_interpolatable_masters(
+        self,
+        designspace,
+        ttf,
+        use_production_names=None,
+        reverse_direction=True,
+        conversion_error=None,
+        feature_writers=None,
+        cff_round_tolerance=None,
+        **kwargs
+    ):
+        if hasattr(designspace, "__fspath__"):
+            ds_path = designspace.__fspath__()
+        if isinstance(designspace, basestring):
+            ds_path = designspace
+        else:
```

Comment:
Is it worth it to test `.path` for `None` and `raise` that fontmake shouldn't be used as a library?

Reply:
Or maybe leave as is for now.